data/acl.go

```diff
@@ -0,0 +1,69 @@
+// Autogenerated jute compiler
+// @generated from '/home/pmazzini/repos/zk/zookeeper.jute'
+
+package data // github.com/go-zookeeper/zk/data
+
+import (
+	"fmt"
+
+	"github.com/go-zookeeper/jute/lib/go/jute"
+)
+
+type ACL struct {
+	Perms int32 // perms
+	Id    *Id   // id
+}
+
+func (r *ACL) GetPerms() int32 {
```

Comment:
The generated code has lots of getters like this. Idiomatic Go avoids getters. Something to watch out for.

Reply:
I think that the thrift compiler does the same thing, I like the fact that it is consistent with it.